doc-turkish/integers.md

```diff
@@ -23,21 +23,21 @@ func TestAdder(t *testing.T) {
 }
 ```
 
-You will notice that we're using `%d` as our format strings rather than `%q`. That's because we want it to print an integer rather than a string.
+Stringleri formatlarken `%q` yerine `%d` kullandığımızı farketmişsinizdir. Bunun sebebi string yerine integer yazdırmak istememiz.
```

Comment:
Burası için **Stringleri** yerine, "Yazıyı" veya "Yazıları" daha uygun olabilir gibi, yinede sana kalmış.

Reply:
Ben Stringler olması gerektiğini düşünüyorum. Eğer Türkçesini kullanırsak **integer** yerie **tam sayı** kullanmamız gerekir. Değişken tipleri vb. olduğu gibi kalması gerektiğini düşünüyorum.

Reply:
Evet evet. Bunu konuştuk, veri tipleri İngilizce olduğu gibi kalmalı.

"You will notice that we're using `%d` as our format **strings** rather than `%q`. That's because we want it to print an integer rather than a string." buradaki strings kelimesi, stringi placeholderlar ile formatlarken yazdığımız "want: %d got %d" gibi olan yazıyı belirtiyor, buda bir string evet ama sadece anlaşılır olması için burada string yerine yazı dersek daha iyi olur gibi geldi.